core/src/com/unciv/logic/automation/unit/WorkerAutomation.kt

```diff
@@ -538,6 +549,17 @@ class WorkerAutomation(
         return distanceBetweenCities + 2 > distanceToEnemy + distanceToOurCity
     }
 
+    /**
+     * Do we want to build a Fort [here][tile] considering surroundings?
+     *
+     * @param  isCitadel Controls within borders check - true also allows 1 tile outside borders
+     * @return Yes the location is good for a Fort here
+     */
+    fun evaluateFortPlacement(tile: Tile, isCitadel: Boolean): Boolean {
+        return evaluateFortSuroundings(tile,isCitadel)
```

Comment:
How about we flip this if? String comparison is much cheaper than the evaluation

Reply:
Done!